puppet/modules/scaleway/templates/manage_snapshots.erb

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/ruby
+#
+# Copyright Foreman Maintainers 2018
+# GPL-3.0
+#
+# This ruby script use the Scaleway API key to
+# ensure that recent snapshots of a volume are taken,
+# and that only the most recent snapshots are kept
+
+SERVER          = <%= @name    -%>
+SETTINGS        = YAML.load_file("/etc/scaleway/<%= @name -%>.yaml")
```

Comment:
I think this should be `/etc/scaleway/#{SERVER}.yaml`

Reply:
Yeah, done